Code/GraphMol/MolOps.h

```diff
@@ -820,14 +821,23 @@ void assignStereochemistryFrom3D(ROMol &mol, int confId = -1,
   \param confId               the conformer to use
 */
 void detectBondStereochemistry(ROMol &mol, int confId = -1);
+void setDoubleBondNeighborDirections(ROMol &mol, const Conformer *conf = NULL);
 
 //! Assign stereochemistry tags to atoms (i.e. R/S) and bonds (i.e. Z/E)
 /*!
-
-  \param mol     the molecule of interest
-  \param cleanIt toggles removal of stereo flags from double bonds that can
-                 not have stereochemistry
-  \param force   forces the calculation to be repeated even if it has
+  Does the CIP stereochemistry assignment for the molecule's atoms
+  (R/S) and double bond (Z/E). Chiral atoms will have a property
+  '_CIPCode' indicating their chiral code.
+
+  \param mol     the molecule to use
+  \param cleanIt if true, atoms with a chiral specifier that aren't
```

Comment:
I don't think this documentation matches what the function does. This function appears to only affect bond directions and not atoms at all.

Reply:
just to make sure we're looking at the same function; this is `assignStereochemistry()`, which certainly operates on both atoms and bonds.

Reply:
Oops! You're correct, the diff was making me think the doc-string came after the declaration of setDoubleBondNeighborDirections.